docs/getting-started/create-block/block-anatomy.md

```diff
@@ -43,19 +42,21 @@ Most of the properties are set in the `block.json` file.
 
 ```json
 {
+	"$schema": "https://schemas.wp.org/trunk/block.json",
```

Comment:
We would have to update it every time a new WP version drops and it might make it incompatible with the Gutenberg plugin. I'm not quite sure how to tackle it.

Reply:
Let's just keep it as is then, it's better to have it set to trunk than not have it at all.